nav2_waypoint_follower/CMakeLists.txt

```diff
@@ -18,9 +23,11 @@ find_package(rclcpp_lifecycle REQUIRED)
 find_package(nav_msgs REQUIRED)
 find_package(nav2_msgs REQUIRED)
 find_package(nav2_util REQUIRED)
+find_package(nav_2d_utils REQUIRED)
```

Comment:
Remove `nav_2d_utils`, thats specific to DWB and I've made sure to isolate it there since it enforces bad assumptions onto the rest of the system. If there's specific functions we need from it, we should add it to `nav2_util` and remove from there. 

(it has alot of SE2 stuff that's bad for full 6DOF motion. I've made sure to isolate it so that everything else in Nav2 other than DWB can handle full 6 DOF navigation assuming 6 DOF algorithms are deployed)